From f93858844e952c234b69a27d6b8118e8df350094 Mon Sep 17 00:00:00 2001
From: fxcoudert <fxcoudert@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Wed, 9 Sep 2009 08:46:32 +0000
Subject: [PATCH] 	PR bootstrap/41180
 	* gcc/cp/Make-lang.in: Remove redundant code from linkage
 	for darwin10.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_4-branch@151555 138bc75d-0d04-0410-961f-82ee72b054a4

[mkl: ported to gcc-4.3.2]
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 gcc/cp/ChangeLog    |    6 ++++++
 gcc/cp/Make-lang.in |    2 +-
 2 files changed, 7 insertions(+), 1 deletion(-)

Index: gcc-4.3.2/gcc/cp/ChangeLog
===================================================================
--- gcc-4.3.2.orig/gcc/cp/ChangeLog
+++ gcc-4.3.2/gcc/cp/ChangeLog
@@ -1,3 +1,9 @@
+2009-09-09  Jack Howarth  <howarth@bromo.med.uc.edu>
+
+	PR bootstrap/41180
+	* gcc/cp/Make-lang.in: Remove redundant code from linkage
+ 	for darwin10.
+
 2008-08-27  Release Manager
 
 	* GCC 4.3.2 released.
Index: gcc-4.3.2/gcc/cp/Make-lang.in
===================================================================
--- gcc-4.3.2.orig/gcc/cp/Make-lang.in
+++ gcc-4.3.2/gcc/cp/Make-lang.in
@@ -73,7 +73,7 @@ g++-cross$(exeext): g++$(exeext)
 CXX_C_OBJS = attribs.o c-common.o c-format.o c-pragma.o c-semantics.o c-lex.o \
 	c-dump.o $(CXX_TARGET_OBJS) c-pretty-print.o c-opts.o c-pch.o \
 	c-incpath.o cppdefault.o c-ppoutput.o c-cppbuiltin.o prefix.o \
-	c-gimplify.o c-omp.o tree-inline.o
+	c-gimplify.o c-omp.o
 
 # Language-specific object files for C++ and Objective C++.
 CXX_AND_OBJCXX_OBJS = cp/call.o cp/decl.o cp/expr.o cp/pt.o cp/typeck2.o \
